<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LZW Compress | Bayram YARIM - 18010011067</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Dictionary Select</string>
    </property>
    <widget class="QRadioButton" name="rdbASCI">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string> ASCII Table</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbUSER">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>User Table</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="teUserTable">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>431</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSelectFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File ...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leFilePath">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>501</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>621</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LZW Compress</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Information</string>
    </property>
    <widget class="QLabel" name="lblInputSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>601</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Input Size :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblOutputSize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>601</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Size :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDiffTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>601</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Different Time : </string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
